projects/plugins/jetpack/extensions/shared/external-media/media-category/index.ts

```diff
@@ -0,0 +1,203 @@
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+
+// Pexels constants
+const PEXELS_ID = 'pexels';
+const PEXELS_NAME = 'Pexels Free Photos';
+const PEXELS_SEARCH_PLACEHOLDER = `Search ${ PEXELS_NAME }`;
+const DEFAULT_PEXELS_SEARCH: MediaSearch = {
+	per_page: 25,
+	search: 'mountain',
+};
+
+// Google Photos constants
+const GOOGLE_PHOTOS_ID = 'google_photos';
+const GOOGLE_PHOTOS_NAME = 'Google Photos';
+const GOOGLE_PHOTOS_SEARCH_PLACEHOLDER = `Search ${ GOOGLE_PHOTOS_NAME }`;
+const DEFAULT_GOOGLE_PHOTOS_SEARCH: MediaSearch = {
+	per_page: 25,
+	search: '',
+};
```

Comment:
Nitpick (non-blocking): I know these will be used by a follow-up PR, but I'd prefer if they are added by that PR.